flags.go

```diff
@@ -209,6 +209,21 @@ func newApp() (app *cli.App) {
 					"inodes.",
 			},
 
+			cli.DurationFlag{
+				Name:  "go-storage-timeout",
```

Comment:
rename this to gcs-call-timeout and other flags just as max-retry-duration and retry-multiplier.

Reply:
fixed